docs/content_management/pages/react_app_block.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Create a block that allows an editor to embed a preconfigured React component into a page.
+---
+
+# React App block
+
+React App block allows an editor to embed a preconfigured React application into a page.
+It is configured in YAML files, under the `ibexa_fieldtype_page` key.
+Page block configuration is not SiteAccess-aware.
+
+Another element of React App Block is `\Ibexa\FieldTypePage\FieldType\Page\Block\Event\Listener\ReactBlock` Listener 
+which adds component and props variables.
+
+It is general for all the blocks.
+
+[[% include 'snippets/page_block_cache_clear.md' %]]
+
+## React App Block configuration
+
+React App blocks are regular [Page blocks](page_blocks.md) and can be confirgured on field definition level as any other block.
+File has exactly the same structure as regular YAML [block configuration](create_custom_page_block/#configure-block), except:
+
+- additional `component` attribute which binds Page Builder block with React App
+- `views` attribute is removed
+
+Each configured React app block has an identifier and the following settings:
+
+|Setting|Description|
+|---|---|
+| `name` | Name of the block used in the Page Builder interface. |
+| `category` | Category in the Page Builder elements menu that the block is shown in. |
+| `thumbnail` | Thumbnail used in the Page Builder elements menu. |
+| `visible` | (Optional) Toggles the block's visibility in the Page Builder elements menu. Remove the block from the layout before you publish another version of the page. |
+| `attributes` | (Optional) List of [block attributes](page_block_attributes.md). |
+
+For example:
+
+``` yaml
+ibexa_fieldtype_page:
+    react_blocks:
+        calculator:
+          name: Calculator (React)
```

Comment:
We have to describe for user that this name should be the same as Component name in /page-builder/react/blocks